build-logic/src/main/kotlin/publishing/maven-utils.kt

```diff
@@ -79,15 +79,12 @@ fun addAdditionalJarContent(project: Project): Unit =
       val additionalJarContent =
         tasks.register("additionalJarContent", Sync::class.java) {
           // Have to manually declare the inputs of this task here on top of the from/include below
-          inputs.files(rootProject.layout.files("LICENSE", "NOTICE"))
+          inputs.files(rootProject.layout.files("licenses/LICENSE", "licenses/NOTICE"))
```

Comment:
I suspect this "clashes" with LICENSE/NOTICE present in src/main/resources/... as well, leading to duplicate and ambiguous files.

Probably necessary to check if any of these files is present in src/main/resources/... and omit the "defaults" here.

But also ensuring that both cases end in the same location in all target jars.

Reply:
Yes, that's actually the behavior I'm proposing:
1. If a specific project (like `polaris-test`) contains `src/main/resources/LICENSE` and `src/main/resources/NOTICE`, then we should use it instead of the default one.
2. If not provided, we should "fallback" to the default ones (in `rootProject` `licenses/LICENSE` and `licenses/NOTICE`)

You are right, I should probably test if the "project specific" `LICENSE` and `NOTICE` files exist to avoid "overwrite/duplicate" resources.